### ISSUE TITLE
NS user load will never load from cache

### ISSUE THREAD

**Issue by mshmsh5000** [DoSomethingArchive/legacy-website#5415]
https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module#L131

`if (!empty($northstar_user_info)) {` below will always evaluate to `FALSE`, because `$northstar_user_info` is not declared.

```
/**
 *Get user data from northstar.
 */
function dosomething_northstar_get_northstar_user($id) {
  if (!empty($northstar_user_info)) {
    $northstar_user_info = cache_get('northstar_user_info_' . $id, 'cache_northstar_user_info');
  } else {
```

**Comment by angaither** [comment]
@chloealee was this fixed with #5408?

**Comment by angaither** [comment]
ignore me, I see now it wasn't merged!
